.eslintrc.json

```diff
@@ -36,6 +36,57 @@
         "args": "after-used",
         "argsIgnorePattern": "^_"
       }
-    ]
-  }
+    ],
+    "import/order": [
+      "error",
+      {
+        "pathGroups": [
+          {
+            "pattern": "{@cowprotocol,@uniswap,@safe-global,@ethersproject,@web3-react}/**",
```

Comment:
unsure if @cowprotocol should belong to this group since its an "sepcial dependency"